util: Add docker environment to run labs (#149)
open-education-hub/operating-systems: util/os-runner/README.md

# OS-runner

> This is an isolated environment to run the examples and tasks from laboratories.

Reviewer: ```suggestion
This is an isolated environment to run the examples and tasks from laboratories and lectures.
```

## Prerequisites

- [Install docker](https://docs.docker.com/get-docker/)

> - Change`OS_HOME` variable from `Makefile` to the directory where you cloned the repo

Reviewer: ```suggestion
- Change the `OS_HOME` variable from `Makefile` to the directory where you cloned the repository
```

## Usage

> `make start` - Start `OS-runner` container
> `make run`   - Attache a new session to `OS-runner` container, this is usefull for tasks requiring multiple terminals
> `make clean` - Stop `OS-runner` container and delete it

Reviewer: ```suggestion
- `make start` - Start `OS-runner` container

- `make run`   - Attache a new session to `OS-runner` container, this is usefull for tasks requiring multiple terminals

- `make clean` - Stop `OS-runner` container and delete it
```

# OS-runner

This is an isolated environment to run the examples and tasks from laboratories and lectures.

## Prerequisites

> - [Install docker](https://docs.docker.com/get-docker/)

Reviewer 2: ```suggestion
- [Install docker](https://get.docker.com/)
```

- Change the `OS_HOME` variable from `Makefile` to the directory where you cloned the repository

## Usage

- `make start` - Start `OS-runner` container

> - `make run`   - Attache a new session to `OS-runner` container, this is usefull for tasks requiring multiple terminals

Reviewer 2: ```suggestion
- `make attach`   - Attach a new session to `OS-runner` container, this is useful for tasks requiring multiple terminals
```
